src/client/src/system/service/operators/getServiceWithMinLoad.ts

```diff
@@ -0,0 +1,45 @@
+import { Observable, Subscription } from 'rxjs'
+import { ServiceInstanceStatus } from '../../../types'
+import LastValueObservableDictionary from './../lastValueObservableDictionary'
+
+export function getServiceWithMinLoad(
+  waitForServiceIfNoneAvailable = true
+): (
+  source: Observable<LastValueObservableDictionary<ServiceInstanceStatus>>
+) => Observable<ServiceInstanceStatus> {
+  return source =>
+    new Observable(obs => {
+      const disposables = new Subscription()
+
+      let findServiceInstanceDisposable = new Subscription()
+
+      disposables.add(findServiceInstanceDisposable)
+
+      findServiceInstanceDisposable = source.subscribe(
```

Comment:
why do you need the two lines above?

Reply:
hmm. Prettier should deal with this stuff.

Reply:
Sorted. This sort of stuff will be auto-fixed on commit

Reply:
I meant, why do we have 

```
let findServiceInstanceDisposable = new Subscription()
disposables.add(findServiceInstanceDisposable)
```

`source.subscribe` returns a subscription which then overrides the findServiceInstanceDisposable variable...